### PR TITLE
feat: copy to clipboard added

### LINKED ISSUE #4511: Group Invite Link "Copy to Clipboard" option
### Is your feature request related to a problem? Please describe

After creating the group we have 2 options to add the numbers
*  entering their emails
*  Invitation link
**My feature request is related to 2nd option where the user needs to copy the invitation link through keys/mouse**

Screenshot from 2024-01-04 04-35-09

### Describe the solution you'd like

In addition to that, we can let the user copy the invitation link just by **clicking a button/icon.**

[Screencast from 01-04-2024 04:40:10 AM.webm](https://github.com/CircuitVerse/CircuitVerse/assets/112646623/d080f1bd-8033-4061-b34d-9cf14366b7f0)

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### Are you working on this?

Yes

### PR DESCRIPTION
**Pull request by Malavi1**
Fixes #4511

#### Describe the changes you have made in this PR -
The complete explanation is found here #4530

### Screenshots of the changes (If any) -
[Screencast from 01-07-2024 01:57:10 PM.webm](https://github.com/CircuitVerse/CircuitVerse/assets/112646623/8c23e756-d3e0-4c60-a684-2ec22f8f88ea)

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/assets/stylesheets/groups.scss [hunk 1]
```diff
@@ -274,10 +274,24 @@
   }
 }
 
-.group-invite-link-text {
+#collapseExample{
```
Comment (tanmoysrt) [inline app/assets/stylesheets/groups.scss:277]: can you pls give it a meaningful-name instead of `collapseExample`

Comment (tanmoysrt) [inline app/assets/stylesheets/groups.scss:277]: can you pls give it a meaningful-name instead of `collapseExample`

**Review by tanmoysrt — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/groups.scss (hunk 1 above)
Comment (Malavi1) [reply app/assets/stylesheets/groups.scss:277]: I just used the `div id` which was defined previously before I make changes. It is okay if I change that!?

Comment (Malavi1) [reply app/assets/stylesheets/groups.scss:277]: @tanmoysrt any inputs !?

**Comment by Malavi1** [comment]
This is the duplicate of #2454

File: app/assets/stylesheets/groups.scss (hunk 1 above)
Comment (tanmoysrt) [reply app/assets/stylesheets/groups.scss:277]: You can provide a better id for this

File: app/views/groups/_add_member_modal.html.erb [hunk 2]
```diff
@@ -39,9 +39,19 @@
...
+          <i class="fas fa-2x fa-regular fa-copy" onclick="copyToClipboard()" id="copyIcon" style="cursor:pointer"></i>
         </div>
       </div>
     </div>
   </div>
 </div>
+<script>
+function copyToClipboard(){
+  var copyText=document.getElementById("embedInviteLink");
+  var copyIcon=document.getElementById("copyIcon);
```
Comment (tanmoysrt) [inline app/views/groups/_add_member_modal.html.erb:52]:
```suggestion
  var copyIcon=document.getElementById("copyIcon");
```

File: app/assets/stylesheets/groups.scss (hunk 1 above)
Comment (tanmoysrt) [reply app/assets/stylesheets/groups.scss]: Updated !

**Review by tanmoysrt — APPROVED** [review]
Looks good :+1:

**Pull request merged**